examples/tox21/inference_tox21.py

```diff
@@ -44,18 +45,29 @@ def main():
     else:
         class_num = len(label_names)
 
-    _, _, test = data.load_dataset(method, labels)
+    _, test, _ = data.load_dataset(method, labels)
+    test = test.get_datasets()
+    X_test = D.NumpyTupleDataset(*test[:-1])
+    y_test = test[-1]
 
     predictor_ = predictor.build_predictor(
         method, config['unit_num'], config['conv_layers'], class_num)
+    chainer.serializers.load_npz(os.path.join(args.in_dir, 'snapshot_iter_92'),
```

Comment:
It is kind to be able to specify model by argument. The default value(in this case, 'snapshot_iter_92') should be set by argparse.